### PR TITLE
Flexible to use more triggers

### PR DESCRIPTION
**Pull request by alokekumardas**

### CODE REVIEW

File: NtupleProducer/plugins/Ntupler.cc [hunk 1]
```diff
@@ -585,6 +671,258 @@ Ntupler::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
             filterName32.push_back(filterName32_allEle);
             filterDecision32.push_back(filterDecision32_allEle);
 
+            // Ele23_Ele12 Trigger matching 
+                        for (unsigned int iteTrigObj = 0 ; iteTrigObj < filterToMatchEle23_Ele12_.size() ; iteTrigObj++)
```
Comment (ram1123) [inline NtupleProducer/plugins/Ntupler.cc]: fix the indentation here and few more places. @alokekumardas

**Review by ram1123 — COMMENTED** [review]
looks fine from my side.

**Comment by arunhep** [comment]
Hi @alokekumardas Thanks for this. Can you please show the snapshots of code successful compilation and running? Did you check whether the branches you added are filled?

File: NtupleProducer/plugins/Ntupler.cc (hunk 1 above)
Comment (alokekumardas) [reply NtupleProducer/plugins/Ntupler.cc]: Thanks for the review I will check all the indentation

**Comment by alokekumardas** [comment]
https://drive.google.com/file/d/1s-dMEJKRACox_z1-MCs3SfHzSm2_Zi4r/view?usp=sharing


Here is the snapshot of successful interactive run with 100 events

**Comment by ram1123** [comment]
> https://drive.google.com/file/d/1s-dMEJKRACox_z1-MCs3SfHzSm2_Zi4r/view?usp=sharing
> 
> Here is the snapshot of successful interactive run with 100 events

@alokekumardas you can attach the screenshot here only. Just drag and drop will work.

**Pull request merged**